admin/spec/service/psychometrician-util.service.spec.js

```diff
@@ -0,0 +1,389 @@
+'use strict'
+/* global describe it expect beforeEach spyOn */
+
+const service = require('../../services/psychometrician-util.service')
+
+// Get a marked check mock
+const checkMockOrig = require('../mocks/check-with-results')
+
+// and a completedCheck that has been marked
+const completedCheckMockOrig = require('../mocks/completed-check-with-results')
+const pupilMockOrig = require('../mocks/pupil')
+const schoolMockOrig = require('../mocks/school')
+
+describe('psychometrician-util.service', () => {
+  let completedCheckMock
+
+  const keyboardInput = [
+    {
+      'question': 1,
+      'clientInputDate': '2017-10-17T18:20:44.447Z',
+      'eventType': 'keydown',
+      'input': '1'
+    },
+    {
+      'question': 1,
+      'clientInputDate': '2017-10-17T18:20:44.558Z',
+      'eventType': 'keydown',
+      'input': '0'
+    },
+    {
+      'question': 1,
+      'clientInputDate': '2017-10-17T18:20:44.678Z',
+      'eventType': 'keydown',
+      'input': 'Enter'
+    }
+  ]
+
+  const touchStartInput = [
```

Comment:
wouldn't it be better if these sets of records were moved to a mock?

Reply:
Yes. Good idea.